soc/nxp/kinetis/ke1xz/Kconfig

```diff
@@ -12,3 +12,9 @@ config SOC_SERIES_KE1XZ
 	select CLOCK_CONTROL
 	select HAS_MCUX
 	select PLATFORM_SPECIFIC_INIT
+
+config SOC_MKE17Z9
+	select HAS_MCUX_LPSPI
```

Comment:
If LPSPI is common to this family then you can put these configs under SOC_SERIES_KE1XZ

Reply:
Thanks for reminding, I will update and put it under SOC_SERIES_KE1XZ.